cluster/images/hyperkube/Dockerfile

```diff
@@ -76,8 +78,11 @@ RUN ln -s /hyperkube /apiserver \
  && ln -s /hyperkube /kubectl \
  && ln -s /hyperkube /kubelet \
```

Comment:
while at it, may be add symlinks in `/usr/local/bin` for these two also? It's kind of annoying these aren't in the path...